CONTRIBUTING.md

```
@@ -157,7 +158,8 @@ For processor code, the following broad rules apply:
    * It may be useful to think: "It's 2am, I've had 2 hours sleep, and there is a bug in prod which is why I'm reading the generated code"...
    * Complex branching logic could also be refactored into a different `RecordVisitor`
  * A gentle preference for for-loops over streams, but emphasis is on *gentle* (i.e. don't make your life hard for it)
- * A gentle preference for `final` modifiers to be used when poissible
+ * A gentle preference for `final` modifiers to be used when possible
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)